File: other/one.java
import java.util.Scanner;
class Main{
    public static void main(String args[])
    {
        int n;
        Scanner s=new Scanner(System.in);
        n=s.nextInt();
        for(int i=0;i<n;i++)
        {
            int d;
            boolean isw;
            d=s.nextInt();
            isw=s.nextBoolean();
            if(d==1 || d==7)
            {
                if(isw)
                System.out.println("9:00");
                else
                System.out.println("6:00");
            }
            else
            {
                if(isw)
                System.out.println("7:00");
                else
                System.out.println("5:00");
            }
        }
        
    }
}

File: other/one.py
class Main:
    n=input();
    i=0
    while(int(i)<int(n)):
        day=int(input());
        isw=bool(input());
        if(day in [1,7]):
            if(isw):
                print("9:00");
            elif(not(isw)):
                print("6:00");
        elif(day in [2,3,4,5,6]):
            if(isw):
                print("7:00");
            else:
                print("5:00");
        i=i+1;

   